testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -325,8 +345,16 @@ public void WriteTestResultsServerRpc(float result, ServerRpcParams receiveParam
     }
 
     [ServerRpc(RequireOwnership = false)]
-    public void WriteErrorServerRpc(string errorMessage, ServerRpcParams receiveParams = default)
+    public void WriteErrorServerRpc(string errorMessage, bool ignoreFormatting = false, ServerRpcParams receiveParams = default)
```

Comment:
@SamuelBellomo @NoelStephensUnity This parameter was added here, affects the message formatting in this method body, and controls whether the message is further formatted with "MPLOG {Date}" as it's passed along to `BaseMultiprocessTests.MultiProcessLog` or if it instead looks like a regular non-multiprocess log... **HOWEVER**, the only caller to this method which sets this boolean is `TriggerActionIdClientRpc`, where the boolean is NOT called `ignoreFormatting` but is instead called `ignoreException`. This seems like a usage mismatch, and that we don't even need this boolean here.

As if that wasn't confusing enough, the only place `ignoreException` in `TriggerActionIdClientRpc` is called, the boolean is actually called `m_IgnoreTimeoutException`, but `TriggerActionIdClientRpc` treats it as a catch-all for ANY `Exception`

To summarize: `IgnoreTimeoutException`->`ignoreException`->`catch(Exception)`->`ignoreFormatting`

long and short of it is... I have no clue what the intention is here of this stuff as a single boolean is used in 4 different places, called 3 different things, for 4 seemingly different purposes. Can someone please clarify what this should actually do?

P.S. THIS IS WHY I HATE BOOLEAN AND DEFAULT PARAMS!

Reply:
@JesseOlmer do you want me to take a crack at resolving this issue? I think this is the last issue pending for this PR.

Reply:
Already fixed and merged into this branch/PR - apologies for leaving this open.

Reply:
It was a bad implementation on my part.